rotkehlchen/chain/substrate/manager.py

```diff
@@ -0,0 +1,555 @@
+        """
+        si_attributes = self.chain.get_substrate_interface_attributes()
+        try:
+            node_interface = SubstrateInterface(
+                url=node.endpoint(),
+                type_registry_preset=si_attributes.type_registry_preset,
+            )
+        except requests.exceptions.RequestException as e:
+            raise RemoteError(
+                f'{self.chain} could not connect to {node} node at '
+                f'endpoint: {node.endpoint()}. Connection error: {str(e)}.',
+            ) from e
+        except (FileNotFoundError, ValueError, TypeError) as e:
+            raise AttributeError('Invalid SubstrateInterface instantiation') from e
+
+        return node_interface
+
+    def _request_chain_api(self, endpoint: Literal['metadata']) -> Response:
+        if endpoint == 'metadata':
+            url = f'{self.chain.get_chain_api()}/scan/metadata'
+        else:
+            raise AssertionError(f'Unexpected {self.chain} endpoint type: {endpoint}')
+
+        log.debug(f'{self.chain} subscan API request', request_url=url)
+        try:
+            response = requests.post(url=url)
+        except requests.exceptions.RequestException as e:
+            raise RemoteError(
+                f'{self.chain} failed to post request at {url}. Connection error: {str(e)}.',
+            ) from e
+
+        return response
+
+    def _request_chain_metadata(self) -> Dict[str, Any]:
+        """Subscan API metadata documentation:
+        https://docs.api.subscan.io/#metadata
+        """
+        response = self._request_chain_api(endpoint='metadata')
+        if response.status_code != HTTPStatus.OK:
+            raise RemoteError(
+                f'{self.chain} chain metadata request was not successful. '
+                f'Response status code: {response.status_code}. '
+                f'Response text: {response.text}.',
+            )
+        try:
+            result = rlk_jsonloads_dict(response.text)
+        except JSONDecodeError as e:
+            raise RemoteError(
+                f'{self.chain} chain metadata request returned and invalid JSON '
+                f'response: {response.text}.',
+            ) from e
+
+        log.debug(f'{self.chain} subscan API metadata', result=result)
+        return result
+
+    def _set_chain_properties(self, node_interface: NodeInterface) -> None:
+        """Return the properties of the chain connected to (e.g. native token,
+        addresses format).
+        """
+        if isinstance(getattr(self, 'chain_properties', None), ChainProperties):
+            log.debug(
+                f'{self.chain} has already the chain properties',
+                chain_properties=self.chain_properties,
+            )
+            return None
+
+        try:
+            self.chain_properties = self._get_chain_properties(node_interface)
+        except RemoteError as e:
+            log.error(str(e))
```

Comment:
Why do this? If you want to log the error, especially without any additional comment just do it at the lowest point where it is raised. And here do nothing and let it bubble up.